### PR TITLE
Correct typos in roadmap/STRUCTURES.md

### PR DESCRIPTION
**Pull request by ShalokShalom**
I would also suggest to add a hint, how structs are used and when and why. 
Currently, this is still obscure to me. An example and some rules of thumb might help.

### CODE REVIEW

File: roadmap/STRUCTURES.md [hunk 1]
```diff
@@ -55,4 +55,4 @@ Would resolve to
 }
 ```
 
-But be advised that using this "polymorhpic" approach strips us from the ability to use type aliases as constructors.
+But be advised that using this "polymorpic" approach strips us from the ability to use type aliases as constructors.
```
[the remark this answers is not in the document]
Comment (ShalokShalom) [reply roadmap/STRUCTURES.md]: Me neither, while dictionary.com is: http://www.dictionary.com/browse/polymorphic

File: roadmap/STRUCTURES.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ type alias Employee x =
      }
 ```
 Which means any struct that has a field `salary` of type integer.
-That way we can define our psuedo-inheritance and polymorphism for more advanced structures.
+That way we can define our pseudo-inheritance and polymorphism for more advanced structures.
```
[the remark this answers is not in the document]
Comment (ShalokShalom) [reply roadmap/STRUCTURES.md]: No, this was the software :P

**Comment by wende** [comment]
@ShalokShalom Any joy?

**Comment by ShalokShalom** [comment]
What do you mean?

**Comment by wende** [comment]
@ShalokShalom I mean the change to `polymorpic` or any further developments in this PR

**Comment by wende** [comment]
@ShalokShalom You introduced a change from `polymorphic` to `polymorpic`. That's not correct as far as I'm aware

**Comment by ShalokShalom** [comment]
I guess both is fine

I showed you the example from the dictionary while I see that both is readable

Maybe such a difference between English UK and US?

File: roadmap/STRUCTURES.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShalokShalom) [reply roadmap/STRUCTURES.md]: Hnn, I did a new typo in this commit xD

**Comment by ShalokShalom** [comment]
I changed it from polymorhpic to polymorpic, which is both wrong :D

Now its correct ^-^

**Comment by wende** [comment]
@ShalokSalam oh! I was reading this issue over 10 times and I've never noticed the h and p were inversed :O
I'm blind

**Comment by ShalokShalom** [comment]
And I committed the wrong thing in the first place, together with a link to the correct one, so I am pretty blind as well ;-)

**Pull request merged**